CHANGELOG.md

```diff
@@ -7,16 +7,57 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.0.0] - 2022-03-30
 
+Amazon Chime SDK JS v3 is here !! 🎉🎉🎉
+
+Amazon Chime SDK for JavaScript v3 includes major improvements for device management, WebRTC metrics, and the 
+messaging session.
+
+- **Device management:** Decouple each user’s audio and video devices from their connection to the WebRTC media 
```

Comment:
nit: How about removing terms like `WebRTC` and `connection`? From the user perspective, they simply join or start a meeting session.

e.g.,

> Decouple audio and video device management from the meeting sessions. For example, a user can select their preferred devices on a video preview page and continue using the same devices to join the session. After joining the session, a user can instantly switch devices without interrupting the ongoing meeting session.